wasm_module/src/result.rs

```diff
@@ -53,7 +53,7 @@ impl Result {
 
         // insert the new character
         self.characters
-            .splice(insert_position..insert_position, [character.clone()]);
```

Comment:
I thought this line would cause trouble, but nothing seems to break. Since I reuse the character in the hot loop (to only have 1 malloc instead of #combiantions amount) the explicit clone.